cluster/up.sh

```diff
@@ -5,7 +5,7 @@ set -ex
 source ./cluster/kubevirtci.sh
 kubevirtci::install
 
-$(kubevirtci::path)/cluster-up/up.sh
+$(kubevirtci::path)/cluster-up/up.sh || $(kubevirtci::path)/cluster-up/kubectl.sh get networkaddonsconfig cluster -o yaml
```

Comment:
The "print" would silence the error here. So setup would continue even after we failed to set it up.

I also think that we could keep the print on the problematic command - on the wait. I don't think it would make sense for other issues that can happen inside "up"

Reply:
Hmm, you're correct about the error being silenced...

As I mentioned before, the wait command does not reside in this repository, it's one of the first two in the following output:
```
~/Sources/macvtap-cni # grep -R kubectl | grep wait 
**_kubevirtci/cluster-provision/k8s-multus/scripts/node01.sh:kubectl --kubeconfig=/etc/kubernetes/admin.conf wait networkaddonsconfig cluster --for condition=Ready --timeout=800s**
**_kubevirtci/cluster-provision/k8s-multus/scripts/provision.sh:kubectl --kubeconfig=/etc/kubernetes/admin.conf wait networkaddonsconfig cluster --for condition=Ready --timeout=800s**
_kubevirtci/cluster-up/cluster/kind/common.sh:    _kubectl wait -n kube-system --timeout=12m --for=condition=Ready -l k8s-app=kube-dns pods
_kubevirtci/cluster-up/cluster/kind/common.sh:    _kubectl wait --for=condition=Ready pod --all -n kube-system --timeout 12m
_kubevirtci/cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh:# not using kubectl wait since with the sriov operator the pods get restarted a couple of times and this is
``` 
It's being run from _kubevirtci witch is `git cloned` at cluster/up.sh runtime.

Reply:
Oh, I'm so sorry for missing that. I though we are creating CNAO here directly. My bad.

In that case, the debug output should go to https://github.com/kubevirt/kubevirtci

Could we limit this PR only to the bump.

Reply:
No problem, removed that commit from this PR